vhdbuilder/packer/test/run-test.sh

```diff
@@ -32,7 +32,7 @@ else
     exit 1
   fi
 fi
-az group create --name $TEST_VM_RESOURCE_GROUP_NAME --location ${AZURE_LOCATION} --tags "source=AgentBaker" "branch=${GIT_BRANCH}"
+az group create --name $TEST_VM_RESOURCE_GROUP_NAME --location ${AZURE_LOCATION} --tags "source=AgentBaker,now=$(date +%s)" "branch=${GIT_BRANCH}"
```

Comment:
`now == created_at`?

Reply:
RGs can be also created by e2e. It's also can be done locally.
Do we need to update e2e tests to include the tag?

Reply:
right now we only target RGs with names matching the regex `test*|vhd-scanning*|pkr-Resource-Group*`, and specifically within the node SIG subscription - e2e RGs/resources are still created in the ACS test subscription which is already onboarded to devinfra's garbage collector

all that to say we shouldn't need to change anything on the e2e side